ansible/roles/nodejs/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Download NVM
+  shell: curl https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh --output nvminstall.sh
+  args:
+    creates: nvminstall.sh
+  become: yes
+  become_user: vagrant
+
+- name: Install NVM
+  shell: sh nvminstall.sh
```

Comment:
Have tried ./ before, unfortunately. Same issue I believe. Testing again to be sure.